README.md

```diff
@@ -1,25 +1,74 @@
+the mainnet game, but that design decision currently makes it difficult to run a playable version
+of the game locally. This repository provides a setup for running a local game with just a few
+steps.
+
+## Builder's Guide
+
+For a full tutorial on using `darkforest-local` with multiple **quests**, check out the [Community Round Builder's Guide](builders_guide.md).
+
+## How darkforest-local Works
+
+This repo uses [submodules](.gitmodules) to pair the [Dark Forest Ethereum
+backend](https://github.com/darkforest-eth/eth) with the [Dark Forest TypeScript
+frontend](https://github.com/darkforest-eth/client) so you can launch a local game.
+
+It also uses Yarn, a package manager and that allows for multiple [workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces) to exist within a project.
+
+The workspaces allows each [submodule](.gitmodules) to have their own packages and configuration. 
+Yarn places all of the packages for each submodule in the top level `node_modules/` folder.
+
+## Requirements
+* Install [Yarn](https://classic.yarnpkg.com/en/docs/install)
+* Install `node >= 14` [nvm](https://github.com/nvm-sh/nvm)
+
+### Fastest Method for Running a Local Game
+1. Fork [darkforest-local](https://github.com/projectsophon/darkforest-local) to your Github repo.
+2. `git clone https://github.com/<your_name>/darkforest-local.git`
+3. `git submodule update --init --recursive --remote --merge`
+4. `yarn`
+5. `yarn start`
+
+### Better Method for Running a Local Game
+1. Fork [darkforest-local](https://github.com/projectsophon/darkforest-local) to your Github repo.
+2. Fork [darkforest-eth/eth](https://github.com/darkforest-eth/eth) to your Github repo
+3. Fork [darkforest-eth/client](https://github.com/darkforest-eth/client) to your Github repo.
+4. Clone your darkforest-local repo: `git clone https://github.com/<your_name>/darkforest-local.git`
+5. Update the `.gitmodules` file to point to your new forks of `eth` and `client.  
+    ex: 
+    * `url = https://github.com/darkforest-eth/eth` =>  
+    `url = https://github.com/cha0sg0d/eth`
+    * `url = https://github.com/darkforest-eth/client` =>  
+    `url = https://github.com/cha0sg0d/client`
+6. Fetch the code from the submodules
+    * `git submodule update --init --recursive`   
+7. Add new branches for developing:
+    - The `darkforest-local` monorepo detaches the submodules from their current HEADs. If you want to save your changes (for example, if you're testing an new contract in `eth`), you'll need to make a new branch in these submodules.
+    1. `cd eth`
+        1. `git checkout -b <new_name>`
+    2. `cd client`
+        1. `git checkout -b <new_name>`
+7. Install packages and dependencies
+    * `yarn`
+8. Start a game
+    * `yarn start`
+
+        If `yarn start` returns the error `ERR_OSSL_EVP_UNSUPPORTED`:  
+        * *This error occurs when using Node v17+*
+        * To fix, run `export NODE_OPTIONS=--openssl-legacy-provider` or switch to `14 <= node <= 16`
```

Comment:
Do we need this? Odd version node aren't recommended for use, as they are canary builds that break all the time.

Reply:
good to know, let's remove